packages/driver/src/primitives/buffer.ts

```diff
@@ -18,41 +18,25 @@
 
 import type char from "./chars";
 import * as chars from "./chars";
+
 import { LegacyHeaderCodes } from "../ifaces";
+import { Buffer } from "node:buffer";
 
 export const utf8Encoder = new TextEncoder();
 export const utf8Decoder = new TextDecoder("utf8");
 
-let decodeB64: (_: string) => Uint8Array;
-let encodeB64: (_: Uint8Array) => string;
-
-if (typeof Buffer === "function") {
-  decodeB64 = (b64: string): Uint8Array => {
-    return Buffer.from(b64, "base64");
-  };
-  encodeB64 = (data: Uint8Array): string => {
-    const buf = !Buffer.isBuffer(data)
-      ? Buffer.from(data.buffer, data.byteOffset, data.byteLength)
-      : data;
-    return buf.toString("base64");
-  };
-} else {
-  decodeB64 = (b64: string): Uint8Array => {
-    const binaryString = atob(b64);
-    const size = binaryString.length;
-    const bytes = new Uint8Array(size);
-    for (let i = 0; i < size; i++) {
-      bytes[i] = binaryString.charCodeAt(i);
-    }
-    return bytes;
-  };
-  encodeB64 = (data: Uint8Array): string => {
-    const binaryString = String.fromCharCode(...data);
-    return btoa(binaryString);
-  };
-}
+const decodeB64 = (b64: string): Uint8Array => {
+  return Buffer.from(b64, "base64") as unknown as Uint8Array;
+};
+
+const encodeB64 = (data: Uint8Array): string => {
+  const buf = !Buffer.isBuffer(data)
+    ? Buffer.from(data.buffer, data.byteOffset, data.byteLength)
+    : data;
+  return buf.toString("base64");
+};
```

Comment:
Hm, I'm not sure about this change -- this is for supporting decodeB64 & encodeB64 in both browser and node, I think... @jaclarke thoughts?

Reply:
Yeah, ideally we shouldn't change this. Is there a reason for importing buffer now?

If I remember we replaced Buffer everywhere with Uint8Array, because node/deno/browser/edge runtime/etc. all support Uint8Array, but Buffer is just a node api (though some bundlers I think automatically inject a polyfill?). Then added this code to use Buffer if it exists as a perf optimisation: https://github.com/edgedb/edgedb-js/issues/674. So I think explicitly importing Buffer again may break some things?

Reply:
oh yes, node import will not work in browser, checking it now

Reply:
Is there a way to CI browser-like environment?

Reply:
There's this test: https://github.com/edgedb/edgedb-js/blob/master/packages/driver/test/browser.test.ts, but I guess it doesn't perfectly replicate a browser environment as it didn't catch this.

Reply:
Hmm, it probably needs updating to disable `node:` prefixed imports as well.

Reply:
Once we get a little distance from this specific task of removing deno, I think we should treat "browser" as another platform and test it the same way we plan on testing deno (and bun, and cloudflare workers, etc) with a _real_ browser via playwright or something similar.

Reply:
+1.